tests/sles4sap/hana_cluster.pm

```diff
@@ -53,7 +53,7 @@ sub run {
 
     if (is_node(1)) {
         # Create the resource configuration
-        my $cluster_conf = get_var('USE_SAP_HANA_SR_ANGI') ? 'angi_hana_cluster.conf' : 'hana_cluster.conf';
+        my $cluster_conf = get_var('USE_SAP_HANA_SR_ANGI') ? 'angi_hana_cluster.conf' : "hana_cluster_$sles4sap::resource_aliase" . ".conf";
```

Comment:
Did you try with
```
my $cluster_conf = get_var('USE_SAP_HANA_SR_ANGI') ? 'angi_hana_cluster.conf' : "hana_cluster_$sles4sap::resource_aliase.conf";
```
I believe that should work.

Reply:
It works. Thanks